src/tests/components/SearchForm.test.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SearchForm from "../../components/searchForm";
+
+describe("SearchForm", () => {
+  const validProps = {
+    searchText: "Topsham",
```

Comment:
Instead of hard coding a value for `searchText` here, why not use `fireEvent` to update the value of the field and mimic the user behaviour in the test?